roles/libvirt_manager/tasks/ocp_layout.yml

```diff
@@ -0,0 +1,66 @@
+---
+## Use a dedicated port, this instance will be removed
+# once OCP cluster is deployed.
+- name: Ensure VBMC is present
+  vars:
+    cifmw_virtualbmc_daemon_port: 51881
+  ansible.builtin.import_role:
+    name: "virtualbmc"
+
+- name: Manage virtual resources
+  vars:
+    cifmw_libvirt_manager_storage_pool: "oooq_pool"
+    action: "create"
+    cifmw_libvirt_manager_pool_dir: "/home/dev-scripts/pool"
+  block:
+    - name: Create storage pool directory
+      become: true
+      ansible.builtin.file:
+        path: "{{ cifmw_libvirt_manager_pool_dir }}"
+        state: directory
+        owner: "{{ ansible_user_id }}"
+
+    - name: Create pool in libvirt
+      ansible.builtin.import_tasks: "storage_pool.yml"
+
+    - name: Craft layout and pass it to create_vms.yml
+      vars:
+        cifmw_libvirt_manager_vm_template: "ocp_domain.xml.j2"
+        cifmw_libvirt_manager_vm_prefix: ''
+        cifmw_libvirt_manager_vm_separator: '_'
+        vm_type: 'ocp_master'
+        _ocp_layout:
+          vms:
+            ocp_master:
+              disksize: "{{ _cifmw_libvirt_manager_layout.vms.ocp.disksize }}"
```

Comment:
the image isn't at that size on the disk - so we shouldn't lose too much space But we might consider a smaller size, so that it's easier to grow later.